chunky/build.gradle

```diff
@@ -14,6 +14,7 @@ configurations {
 dependencies {
   implementation 'it.unimi.dsi:fastutil:8.4.4'
   implementation 'org.apache.commons:commons-math3:3.2'
+  implementation 'org.apache.commons:commons-collections4:4.4'
```

Comment:
Reminder to self: I'll have to add this library to the `latest.json` and `snapshot.json` and host it.